dipy/reconst/tests/test_ivim.py

```diff
@@ -11,101 +11,84 @@
        of brain perfusion with intravoxel incoherent motion
        MR imaging." Radiology 265.3 (2012): 874-881.
 """
-import warnings
 import numpy as np
 from numpy.testing import (assert_array_equal, assert_array_almost_equal,
                            assert_raises, assert_array_less, run_module_suite,
-                           assert_, assert_equal, dec)
-from dipy.testing import assert_greater_equal
+                           dec)
 
 from dipy.reconst.ivim import ivim_prediction, IvimModel
 from dipy.core.gradients import gradient_table, generate_bvecs
 from dipy.sims.voxel import multi_tensor
 
 from dipy.utils.optpkg import optional_package
-
 cvxpy, have_cvxpy, _ = optional_package("cvxpy")
+
 needs_cvxpy = dec.skipif(not have_cvxpy)
 
 
-def setup_module():
```

Comment:
Please, do not remove this function `setup_module()`. Can you update this function instead of coming back to an old version. Thank you